product_unique_serial/tests/test_for_unicity.py

```diff
@@ -30,6 +30,20 @@
 from openerp.tests.common import TransactionCase
 from openerp.tools import mute_logger
 from psycopg2 import IntegrityError
+from openerp import _
+
+note = _(
+    u'Remember: When a serial number (lot) is selected, its quantity '
+    u'is fixed against the quantity in the serial number (lot) and not '
+    u'against the quantity in full of the product.')
+
+msg_greater = _(
+    u'Product %s has been configured to use unique lots. '
+    u'You are trying to set %s items in lot %s. %s')
+
+msg_increase = _(
+    u'Product %s has been configured to use unique lots. '
+    u'You are trying to increase %s items in the lot %s. %s')
```

Comment:
Avoid global variables.

Can you please add them as attributes in the class and they will be available in all the class, use this kind of variables is a bad practice.